app/products/calls/connection/websocket_event_handlers.ts

```diff
@@ -228,3 +232,26 @@ export const handleHostLowerHand = async (serverUrl: string, msg: WebSocketMessa
 
     unraiseHand();
 };
+
+export const handleHostRemoved = async (serverUrl: string, msg: WebSocketMessage<HostControlsMsgData>) => {
+    const currentCall = getCurrentCall();
+    if (currentCall?.serverUrl !== serverUrl ||
+        currentCall?.channelId !== msg.data.channel_id ||
+        currentCall?.mySessionId !== msg.data.session_id) {
+        return;
+    }
+
+    leaveCall();
```

Comment:
Just curious, was it unfeasible to pass an error to `leaveCall()` and re-use the existing logic to show alerts (`showErrorAlertOnClose`)?

Reply:
I initially didn't want to do that because being removed by the host isn't an error state... But we're already stretching the meaning of error to pass an alert in the other cases, so I guess it follows that. Thanks!